### PR TITLE
Web app improvements

### PR DESCRIPTION
**Pull request by tsteenbe**
See individual commits for details.

### CODE REVIEW

File: reporter-web-app/config/webpack.config.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const safePostCssParser = require('postcss-safe-parser');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
```
Comment (mnonnenmacher) [inline reporter-web-app/config/webpack.config.js:16]: Commit message:
* No need to repeat "plugin" in the title.
* Use present tense in the body, "Removing plugin..." -> " Remove the plugin..."

File: reporter-web-app/src/components/PackageFindingsTable.js [hunk 2]
```diff
@@ -19,14 +19,104 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Table } from 'antd';
-import { CopyrightOutlined, FileTextOutlined } from '@ant-design/icons';
+import { Table, Tooltip } from 'antd';
```
[the remark this answers is not in the document]
Comment (tsteenbe) [reply reporter-web-app/src/components/PackageFindingsTable.js:22]: Fixed "support" and added an explanation of what support means in the commit message.

File: reporter-web-app/config/webpack.config.js (hunk 1 above)
Comment (tsteenbe) [reply reporter-web-app/config/webpack.config.js:16]: Fixed.

File: reporter-web-app/src/components/TableView.js [hunk 3]
```diff
@@ -194,6 +194,8 @@ class TableView extends React.Component {
                     </span>
                 ),
                 responsive: ['md'],
+                sorter: (a, b) => a.declaredLicenses.size - b.declaredLicenses.size,
```
Comment (sschuberth) [inline reporter-web-app/src/components/TableView.js:197]: Commit message: Please add a dot after "Closes #2606" and a blank line to separate it from the footer.

File: reporter-web-app/config/webpack.config.js (hunk 1 above)
Comment (sschuberth) [reply reporter-web-app/config/webpack.config.js:16]: The body still says "Removing plugin..."

File: reporter-web-app/src/components/PackageFindingsTable.js [hunk 4]
```diff
@@ -19,14 +19,104 @@
...
+import {
```
Comment (fviernau) [inline reporter-web-app/src/components/PackageFindingsTable.js:23]: commit message: The title says "package configuration excludes", but isn't the handling actually for `PathExcludes` in general, that is for `packages` and `projects`?

How about instead: "reporter-web-app: Handle path excludes on finding granularity"
as previously they have been handled only on project granularity.

File: reporter-web-app/src/components/TableView.js (hunk 3 above)
Comment (tsteenbe) [reply reporter-web-app/src/components/TableView.js:197]: Fixed.

File: reporter-web-app/config/webpack.config.js (hunk 1 above)
Comment (tsteenbe) [reply reporter-web-app/config/webpack.config.js:16]: Fixed

File: reporter-web-app/src/components/PackageFindingsTable.js (hunk 4 above)
Comment (tsteenbe) [reply reporter-web-app/src/components/PackageFindingsTable.js:23]: Fixed

**Pull request merged**